docs/typescript.md

```diff
@@ -1,29 +1,76 @@
 ---
 layout: default
-title: '[EXPERIMENTAL] Using TypeScript with DNSControl'
+title: Using TypeScript with DNSControl
 ---
 
-# [EXPERIMENTAL] Using TypeScript with DNSControl
+# Using TypeScript with DNSControl (experimental)
 
-> **NOTE**: This feature is currently experimental. Type information may change in future releases, and any release (even a patch release) could introduce new type-checking errors into your `dnsconfig.js` file. If you plan to use TypeScript, please file bug reports for any issues you encounter, and avoid manually running type-checking as part of your deployment process until the “experimental” label is removed.
+## What is this?
 
-While DNSControl does not support TypeScript syntax in `dnsconfig.js`, you can still use TypeScript’s features in editors which support it (including Visual Studio Code). To set up TypeScript support in Visual Studio Code, follow these steps:
-First, you’ll need to grab the `dnscontrol.d.ts` file corresponding to your version of DNSControl. You can manually download this file off of GitHub, or you can use the following command:
+Would you like your editor to support auto-completion and other advanced IDE
+features when editing dnsconfig.js?  Yes you can!
+
+While DNSControl does not support TypeScript syntax in `dnsconfig.js`, you can
+still use TypeScript’s features in editors which support it.
+
+If you’re using VS Code (or another editor that supports TypeScript), you
+should now be able to see the type information in your `dnsconfig.js` file as
+you type. Hover over record names to read their documentation without having to
+open the website!
+
+## How to activate
+
+To set up TypeScript support in Visual Studio Code, follow these steps:
```

Comment:
How about other IDE? Would you like to describe them all? For example the [JetBrains PHStorm documentation](https://www.jetbrains.com/help/phpstorm/typescript-support.html) is very complete and makes for me no sense to copy/paste all there content. What do you think?

Reply:
These steps should be the same for all editors.